ert_shared/storage/storage_api.py

```diff
@@ -288,24 +286,73 @@ def ensemble_schema(self, ensemble_id):
                         )
                     ],
                     "parameters": [
-                        {"name": par.name}
+                        self._parameter_minimal(
+                            name=par.name,
+                            group=par.group,
+                            prior=par.prior,
+                            parameter_def_id=par.id,
+                        )
                         for par in rdb_api.get_parameter_definitions_by_ensemble_id(
                             ensemble_id
                         )
                     ],
-                    "priors": [
-                        {
-                            "group": prior.group,
-                            "key": prior.key,
-                            "function": prior.function,
-                            "parameter_names": prior.parameter_names,
-                            "parameter_values": prior.parameter_values,
-                        }
-                        for prior in ens.priors
-                    ]
-                    if ens.priors is not None
-                    else [],
                 }
             )
 
             return return_schema
+
+    def _parameter_minimal(self, name, group, prior, parameter_def_id):
+        return {
+            "key": name,
+            "group": group,
+            "parameter_ref": parameter_def_id,
+            "prior": {
+                "function": prior.function,
+                "parameter_names": prior.parameter_names,
+                "parameter_values": prior.parameter_values,
+            }
+            if prior is not None
+            else {},
+        }
+
+    def parameter(self, ensemble_id, parameter_def_id):
```

Comment:
i would prefer naming like get_parameter since this is an api on top of the database

Reply:
As of now the naming convention has mainly been like this At some point, we have to go over the storage_api and write a consistent api, and I agree that a get_ prefix might be nicer to read.

Reply:
Create separate issue and merge as is?

Reply:
I would say merge as is and it will be part of the code cleanup issue :)

Reply:
Added here: https://github.com/equinor/ert/issues/819